### PR TITLE
Grammar improvements

### PR DESCRIPTION
**Pull request by kadenscott**
Just a few tweaks to grammar, let me know if there's anything you'd like to change. Maybe keep this open if I find anything else, tomorrow (when I have some time) I'll take a deeper look.

### CODE REVIEW

File: src/main/frontend/src/assets/locales/en.json [hunk 1]
```diff
@@ -427,29 +427,29 @@
       },
       "keywords": {
         "_": "Keywords",
-        "info": "These are special words that will return your project when people add them to their searches. Max 5."
+        "info": "These are special words that will return your project when people add them to their searches. (max 5 keywords)"
```
Comment (KennyTV) [inline src/main/frontend/src/assets/locales/en.json:430]:
```suggestion
        "info": "These are special words that will return your project when people add them to their searches (max 5 keywords)."
```

File: src/main/frontend/src/assets/locales/en.json [hunk 2]
```diff
@@ -427,29 +427,29 @@
...
       "_": "Settings",
       "deployKey": {
         "_": "Deployment key",
-        "info": "Generate a unique deployment key to enable build deployment from Gradle"
+        "info": "Generate a unique deployment key to enable build deployment from Gradle."
       },
       "description": {
         "_": "Description",
-        "info": "A short description of your project (max 120)"
+        "info": "A short description of your project. (max 120 characters)"
```
Comment (KennyTV) [inline src/main/frontend/src/assets/locales/en.json:443]:
```suggestion
        "info": "A short description of your project (max 120 characters)."
```

File: src/main/frontend/src/assets/locales/en.json [hunk 3]
```diff
@@ -427,29 +427,29 @@
...
       },
       "forumSync": {
         "_": "Create posts on the forums",
-        "info": "Sets if events like a new release should automatically create a post on the forums"
+        "info": "Should events (like new releases) have a forum post automatically created?"
```
Comment (KennyTV) [inline src/main/frontend/src/assets/locales/en.json:447]:
```suggestion
        "info": "Should events like new releases have a forum post created automatically?"
```

**Review by KennyTV — CHANGES_REQUESTED** [review]

File: src/main/frontend/src/assets/locales/en.json (hunk 3 above)
Comment (mdcfe) [inline src/main/frontend/src/assets/locales/en.json:447]:
```suggestion
        "info": "Should events like new releases automatically create a forum post?"
```
@KennyTV Slightly different and reads better to me, not sure what others think though

**Comment by Machine-Maker** [comment]
I’m not sure about this? You are changing the grammar in the frontend json file. We have identical localization in the hangar app resources folder “messages.properties”. Every so often, I’ve just been using a script I wrote to convert from .properties to .json. So at some point, we need better integration of those two systems. 


TLDR; make changes to messages in “message.properties” NOT the json since that will get generated automatically from messages.properties.

**Comment by kadenscott** [comment]
Lol, sounds good. Sorry, still have yet to figure out how this project is structured. I'm gonna delete & re-fork and clone it so I can properly see what I'm doing. Will open a new PR in a few.
